src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/GoogleDataprocDAO.scala

```diff
@@ -397,10 +403,18 @@ class GoogleDataprocDAO(protected val leoServiceAccountEmail: WorkbenchEmail,
         }
 
         val transformed: OptionT[Future, Unit] = for {
-          dCluster <- OptionT.liftF[Future, DataprocCluster] { getCluster(cluster.googleProject, cluster.clusterName) }
-          stagingBucket <- OptionT.fromOption { getStagingBucket(dCluster) }
-          _ <- OptionT.liftF[Future, BucketAccessControl] { executeGoogleRequestAsync(cluster.googleProject, s"Bucket ${stagingBucket.name}", bacInsert(stagingBucket)) }
-          _ <- OptionT.liftF[Future, ObjectAccessControl] { executeGoogleRequestAsync(cluster.googleProject, s"Bucket ${stagingBucket.name}", doacInsert(stagingBucket)) }
+          dCluster <- OptionT.liftF[Future, DataprocCluster] {
+            getCluster(googleProject, clusterName)
+          }
+          stagingBucket <- OptionT.fromOption {
+            getStagingBucket(dCluster)
+          }
+          _ <- OptionT.liftF[Future, BucketAccessControl] {
+            executeGoogleRequestAsync(googleProject, s"Bucket ${stagingBucket.name}", bacInsert(stagingBucket))
+          }
+          _ <- OptionT.liftF[Future, ObjectAccessControl] {
+            executeGoogleRequestAsync(googleProject, s"Bucket ${stagingBucket.name}", doacInsert(stagingBucket))
+          }
```

Comment:
i think the old whitespacing was more readable here

Reply:
+1, these monad transformers are annoyingly verbose at times, but I usually try to keep for-comp `<-`s on 1 line if possible.